features/flows/elastic/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,6 +18,8 @@
             <cm:property name="elasticUser" value="" />
             <cm:property name="elasticPassword" value="" />
             <cm:property name="elasticIndexStrategy" value="monthly" />
+            <cm:property name="defaultMaxTotalConnectionPerRoute" value="-1" />
+            <cm:property name="maxTotalConnection" value="-1" />
```

Comment:
I can add those in the docs in HZN-1178. Does -1 mean, there is no upper limit?

Reply:
See https://github.com/OpenNMS/opennms/pull/1752/files#diff-89d0a529573b1b0f14a653bc351e56e4R159